features/pim-custom-fields.feature

```diff
@@ -0,0 +1,30 @@
+Feature: PIM Module Custom Fields capability
+
+  Scenario Outline: Add a custom dropdown field to PIM
+    Given Admin is on Custom Fields
+    When Admin adds a new DropDown Custom Field
+      | Field Name   | Screen   | Type     | Select Options |
+      | <fieldName>  | <screen> | Dropdown | <options>     |
```

Comment:
Is this syntax correct? (genuine question, I haven't done this in Playwright/TS)